docs/GHAFileExplained.md

```diff
@@ -116,6 +116,11 @@ jobs:
       #   IGNORE_PATHS: 'ignore'
       #   IGNORE_NAMES: 'ignore_name.php'
       #   MUSTACHE_IGNORE_NAMES: 'broken.mustache'
+      #
+      # You also can specify specific command ignores too:
+      #   CODECHECKER_IGNORE_PATHS: 'ignore'
+      #   CODECHECKER_IGNORE_NAMES: 'ignore_name.php'
+      # For more information, see IgnoringFiles.md.
```

Comment:
`For more information` line I would suggest to remove, instead refer to Help page in the line 112. e.g. "For more details on configuring for specific requirements please refer to Help page." Help page contains link to ignore files and other cases such as coverage, plugin dependencies, etc.